pom.xml

```diff
@@ -71,7 +71,7 @@
             ${project.basedir}/../coverage-report/target/site/jacoco-aggregate/jacoco.xml
         </sonar.coverage.jacoco.xmlReportPaths>
         <!-- Axon Server -->
-        <axonserver-connector-java.version>2023.2.0</axonserver-connector-java.version>
+        <axonserver-connector-java.version>2024.0.0-SNAPSHOT</axonserver-connector-java.version>
```

Comment:
We should remember to put in a released version, at the latest before releasing `4.10` of framework. Ideally through it's better to change it to the released version as soon as it's available, still in this pr.

Reply:
The release plugin will automatically detect and resolve dependencies pointing at snapshots.